### PR TITLE
New tool to make even calling intervals from a set of gvcfs

### PR DESCRIPTION
**Pull request by eitanbanks**
### Description

This is a tool that attempts to make an even calling interval list from a set of input gvcfs.

The motivation is that the size of the inputs (rather than the size of the output vcfs) is what's most correlated to the runtime of any given shard for calling.  So this tool keeps track of all the potential variant positions within a set of gvcfs and splits up the genome into mostly even chunks across all of those positions.

I have no idea whether this is useful to anyone else.  If not, I'll be fine if this never gets merged.

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [ ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

### CODE REVIEW

File: src/main/java/picard/vcf/CreateEvenIntervals.java [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import java.util.*;
+
+/**
+ * Traverses one or more gVCF files in order to generate a list of intervals that have approximately
+ * the same number of non-reference positions in all of them.
+ */
+@CommandLineProgramProperties(
+        usage = "Traverses one or more gVCF files in order to generate a list of intervals that have approximately " +
+                "the same number of non-reference positions in all of them.  Assumes files contain sequence dictionaries " +
+                "(and that they are all the same).",
```
Comment (gbggrant) [inline src/main/java/picard/vcf/CreateEvenIntervals.java]: It's easy enough to check this with
dict.assertSameDictionary(in.getFileHeader().getSequenceDictionary())

**Comment by eitanbanks** [comment]
I've addressed comments from both @nh13 and @gbggrant.  Does that qualify as a thumbs up?

**Comment by yfarjoun** [comment]
no. you need an actual thumbs up. otherwise these might be comments from a glance.

**Comment by yfarjoun** [comment]
I'm not sure that this is useful for anyone that isn't making giant callsets....unless someone asks for it, I think it should belong in the private repo.

**Comment by kbergin** [comment]
To clean up this repo, we are deleting stale branches that have been merged or closed. You have a short period of time to restore this branch if needed. Thanks!

**Pull request closed without merging**